phpunit/class-wp-theme-json-test.php

```diff
@@ -28,7 +28,7 @@ class WP_Theme_JSON_Gutenberg_Test extends WP_UnitTestCase {
 	 *
```

Comment:
All the changes in this file are whitespace changes between property and value in CSS declarations.

The Style Engine outputs rules without whitespace, e.g., `font-size:60px;`, whereas the theme json class outputs `font-size: 60px;`